test/src/api/protocol/testElementSelectors.js

```diff
@@ -0,0 +1,731 @@
+var nock = require('nock');
```

Comment:
would it be possible to split these tests over several files and also keeps the nocks separate?